src/viewer.js

```diff
@@ -23,6 +24,8 @@ module.exports = {
   start: startServer
 };
 
+const title = `${process.env.npm_package_name || 'Webpack Bundle Analyzer'} [${utils.getCurrentTime()}]`;
```

Comment:
Who sets `npm_package_name` env variable? Is it `npm`?
Does it mean that it will be empty if I run webpack via `gulp`?

Reply:
Yes, `npm` set env variable. 
We can use `npm scripts` to run webpack via `gulp`.

Or use `fs` to `readFile`? Feel kind of weird.
I think this is fine. We have default name after all.